packages/sparrow-wifi/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=sparrow-wifi
+pkgver=197.60e9ce7
+pkgrel=1
+pkgdesc='Next-Gen GUI-based WiFi and Bluetooth Analyzer.'
+arch=('any')
+groups=('blackarch' 'blackarch-bluetooth' 'blackarch-wireless')
+url='https://github.com/ghostop14/sparrow-wifi'
+license=('GPL-3.0')
```

Comment:
this is Deprecated License Identifiers, it should be GPL-3.0-only or  GPL-3.0-or-later, see https://spdx.org/licenses/